DigitalLearningSolutions.Data/DataServices/LearningResourceReferenceDataService.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace DigitalLearningSolutions.Data.DataServices
+{
+    using System.Data;
+    using System.Linq;
+    using Dapper;
+
+    public interface ILearningResourceReferenceDataService
+    {
+        int GetLearningHubResourceReferenceById(int learningResourceReferenceId);
+    }
+
+    public class LearningResourceReferenceDataService : ILearningResourceReferenceDataService
+    {
+        private readonly IDbConnection connection;
+
+        public LearningResourceReferenceDataService(IDbConnection connection)
+        {
+            this.connection = connection;
+        }
+
+        public int GetLearningHubResourceReferenceById(int learningResourceReferenceId)

```

Comment:
I'm not sure I like the name of this method, but I don't think there are any good alternatives. The only way I think this could be nicer is if it returned the full record data and was called "GetLearningResourceReferenceById" but that seems overkill to me. Thoughts?

Reply:
I don't see a reason to get back the rest of the record at the moment. I think the only other reasonable names would be GetResourceRefIdByLearningResourceReferenceId or GetLearningHubResourceReferenceByLearningResourceReferenceId